sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -391,14 +403,14 @@ The following section provides several code snippets illustrating common pattern
 These code samples show common scenario operations with the Azure Form Recognizer client library.
 The async versions of the samples show asynchronous operations with Form Recognizer.
 
-* Recognize receipts: [RecognizeReceipts][recognize_receipts] ([async][recognize_receipts_async])
-* Recognize receipts from a URL: [RecognizeReceiptsFromUrl][recognize_receipts_from_url] ([async][recognize_receipts_from_url_async])
-* Recognize content: [RecognizeContent][recognize_content] ([async][recognize_content_async])
-* Recognize custom forms: [RecognizeCustomForms][recognize_custom_forms] ([async][recognize_custom_forms_async])
-* Train a model without labels: [TrainModelWithoutLabels][train_unlabeled_model] ([async][train_unlabeled_model_async])
-* Train a model with labels: [TrainModelWithLabels][train_labeled_model] ([async][train_labeled_model_async])
-* Manage custom models: [ManageCustomModels][manage_custom_models] ([async][manage_custom_models_async])
-* Copy a model between Form Recognizer resources: [CopyModel][copy_model] ([async][copy_model_async])
+* Recognize receipts: [RecognizeReceipts][recognize_receipts] ([RecognizeReceiptsAsync][recognize_receipts_async])
```

Comment:
It still looks like the same line. What about making it a table? so sync and async links in different columns?
Or do a subsection or sync and another for async?